src/Typeahead.react.js

```diff
@@ -338,11 +342,17 @@ const Typeahead = React.createClass({
       minLength,
       multiple,
       name,
-      placeholder,
       renderToken,
     } = this.props;
     const {activeIndex, activeItem, initialItem, selected, text} = this.state;
     const Input = multiple ? TokenizerInput : TypeaheadInput;
+    let placeholder = '';
+    if(multiple) {
+      placeholder = selected.length > 0 ? this.props.multiplePlaceholder :
```

Comment:
This feels unnecessary to me since it's probably a narrow use case. If someone really wants to display a different string, they can already do something like:

```jsx
<Typeahead
  multiple
  onChange={selected => this.setState({selected})}
  options={...}
  placeholder={this.state.selected.length ? 'I have a selection' : 'I have no selections'}
/>
```